db/migrations/20191203104431_create_historical_urn_state_computed_columns.sql

```diff
@@ -0,0 +1,46 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+
+CREATE FUNCTION api.historical_urn_state_ilk(state api.historical_urn_state)
+    RETURNS api.ilk_state AS
+$$
+SELECT *
+FROM api.get_ilk(state.ilk_identifier, state.block_height)
```

Comment:
What's the complexity of this call? And the rest really...

Might be a trade-off to be done here, fast access to the "native" data, but having to do a second query to grab the `ilk`/`frobs`/`bites` for a certain block... Not sure actually.

Reply:
But my point is that if these slow down the queries by a lot, maybe it's not worthwhile to include them even...

Reply:
Yeah that's a good point. I just included these computed columns because they're included in the original `all_urn_states` query, and I wanted to match that functionality. It's probably worth doing some performance analysis on these columns, bc just from looking at the `get_ilk` query it seems pretty complex.

Reply:
Yeah, `get_ilk` is what usually locks the endpoint completely, so it's gonna nuke this query too :)